Application/Features/VacationFeature/Commands/CreateVacationCommand.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Application.Exceptions;
+using Application.Interfaces;
+using AutoMapper;
+using Domain.Models;
+using FluentValidation;
+using MediatR;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+
+namespace Application.Features.VacationFeature.Commands
+{
+    public class CreateVacationCommandRequest : IRequest<CreateVacationCommandResponse>
+    {
+        public int UserId { get; set; }
+        public DateTimeOffset VacationStart { get; set; }
+        public DateTimeOffset VacationEnd { get; set; }
+    }
+
+    public class CreateVacationCommandValidator : AbstractValidator<CreateVacationCommandRequest>
+    {
+        public CreateVacationCommandValidator()
+        {
+            RuleFor(x => x.UserId).NotEmpty().WithMessage("UserId must not be blank");
```

Comment:
at least end date should be greater that start date

Reply:
Done